model_destinations.go

```diff
@@ -142,6 +142,25 @@ func getDestinationByLockingScript(ctx context.Context, lockingScript string, op
 	return destination, nil
 }
 
+// getDestinations will get all the destinations with the given conditions
+func getDestinations(ctx context.Context, metadata *Metadata, conditions *map[string]interface{},
+	queryParams *datastore.QueryParams, opts ...ModelOps) ([]*Destination, error) {
+
+	modelItems := make([]*Destination, 0)
+	if err := getModelsByConditions(ctx, ModelDestination, &modelItems, metadata, conditions, queryParams, opts); err != nil {
+		return nil, err
```

Comment:
Does opts need the `...`

`opts` vs `opts...`

Not sure if that is an error or not...